app/assets/stylesheets/components/_members.scss

```diff
@@ -0,0 +1,59 @@
+.members {
+  h3 {
+    text-align: center;
+    font-size: 2rem;
+    font-weight: 400;
+    margin: $base-spacing 0;
+
+    .badge {
+      font-size: .8rem;
+    }
+  }
+}
+
+.members-grid {
+  text-align: center;
+
+  p {
+    position: absolute;
+    vertical-align: middle;
+    background-color: rgba($black-color, 0.6);
+    line-height: 150px;
+    color: $white-color;
+    opacity: 0;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+    font-size: .9rem;
+  }
+
+  a {
+    color: $white-color;
+    border: 0;
+    margin: 5px;
+    display: inline-block;
+    background-size: cover;
+    position: relative;
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+    -webkit-border-radius: 50%;
+    overflow: hidden;
+    opacity: .8;
+
+    &, p {
+      @include transition(opacity 0.5s);
+    }
+
+    &:hover,
+    &:focus,
+    &:active {
+      &, p {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line